app/src/main/java/com/example/photostomusic/fragments/CameraFragment.java

```diff
@@ -115,22 +78,38 @@ public void onClick(View v) {
             }
         });
 
+        Bundle bundle = this.getArguments();
+        spotifyToken = bundle.getString("key");
+
+
+
+        // Hashmap used to relate emotions to music genres
+        // Emotions and music relation based on the paper:
+        // https://www.pnas.org/content/117/4/1924#sec-3
+        // Its results can be easily seen on this interactive
+        // cluster: https://www.ocf.berkeley.edu/~acowen/music.html#
+        emotionRelation = new HashMap<String, String>() {{
+            put("Calm", "acoustic");
+            put("Dreamy", "gospel");
+            put("Heroic", "classical");
+            put("Anxious", "industrial");
+            put("Scared", "sad");
+            put("Annoyed", "punk");
+            put("Defiant", "metal");
+            put("Energized", "edm");
+            put("Amazed", "funk");
+            put("Joyful", "disco");
+            put("Desirous", "reggaeton");
+            put("Cute", "romance");
```

Comment:
oh this is really interesting!

Reply:
It is! I enjoyed a lot looking for papers to fundament my claims